Library/Homebrew/env_config.rb

```diff
@@ -0,0 +1,298 @@
+# frozen_string_literal: true
+
+require "os"
+require "hardware"
```

Comment:
This is problematic. These files will end up using `Homebrew::EnvConfig` like in `Library/Homebrew/extend/os/linux/hardware/cpu.rb`.